standalone/gradle.properties

```diff
@@ -1,12 +1,13 @@
 # Location of the repository for various artifacts used in the build
-artifactoryContextUrl=https://artifactory.labkey.com/artifactory
+artifactory_contextUrl=https://artifactory.labkey.com/artifactory
 
 # Version 1.17.0 or higher is required to fully support building stand-alone modules with the LabKey Gradle plugins
-gradlePluginsVersion=1.17.0
+# Exact version will vary based on the version of LabKey being built on
+gradlePluginsVersion=1.27.0_standaloneBuild-SNAPSHOT
 
 # versions of artifacts required as dependencies
-labkeyVersion=20.7.0
-labkeyClientApiVersion=1.3.0
+labkeyVersion=21.3-SNAPSHOT
```

Comment:
I think we want this to be a release version since that provides the most stability for doing external development.